src/connection_transfer/connection_transfer.go

```diff
@@ -15,10 +14,21 @@ const (
 	CONNECTION_TRANSFER_HANDLER_NAME = "connectionTransfer"
 )
 
+type ConnectionTransferConfig struct {
+	UpdateTopic string `toml:"update_topic"`
+}
+
 var (
+	connectionTransferConfig = ConnectionTransferConfig{
+		UpdateTopic: "connection/update",
+	}
 	connectionTransfer *ConnectionTransfer
 )
 
+func SetConfig(config ConnectionTransferConfig) {
```

Comment:
you don't need one of these functions if you make config global, it also ensures you dont forget to call it